src/components/atoms/charts/Calendar3D/index.jsx

```diff
@@ -0,0 +1,374 @@
+    </span>\n
+    </div>`,
+    };
+  }
+
+  renderBottomStats() {
+    let streakLongest, datesLongest, streakCurrent, datesCurrent, contribData;
+
+    if (this.props.year) {
+      contribData = this.props.platformData.statistic.years.find(
+        (element) => element.year === this.props.year
+      );
+    } else {
+      contribData = this.props.platformData.statistic.current;
+    }
+
+    let contributionCalendar = contribData.calendar;
+
+    if (contribData.streak.longest) {
+      streakLongest = contribData.streak.longest.totalDays;
+      datesLongest =
+        moment(contribData.streak.longest.startDate).format("MMM DD, YYYY") +
+        " - " +
+        moment(contribData.streak.longest.endDate).format("MMM DD, YYYY");
+    } else {
+      streakLongest = "0";
+      datesLongest = "-";
+    }
+    if (contribData.streak.current.id !== -1) {
+      streakCurrent = contribData.streak.current.totalDays;
+      datesCurrent =
+        moment(contribData.streak.current.startDate).format("MMM DD, YYYY") +
+        " - " +
+        moment(contribData.streak.current.endDate).format("MMM DD, YYYY");
+    } else {
+      streakCurrent = "0";
+      datesCurrent = "-";
+    }
+
+    return {
+      __html: `<div class="ic-stats-block ic-stats-bottom">\n
+    <span class="ic-stats-table">\n
+    <span class="ic-stats-row">\n
+    <span class="ic-stats-label">Longest streak\n
+    <span class="ic-stats-count">${streakLongest}</span>\n
+    </span>\n
+    <span class="ic-stats-meta">\n
+    <span class="ic-stats-unit">days</span>\n
+    <span class="ic-stats-date">${datesLongest}</span>\n
+    </span>\n
+    </span>\n
+    <span class="ic-stats-row">\n
+    <span class="ic-stats-label">Current streak\n
+    <span class="ic-stats-count">${streakCurrent}</span>\n
+    </span>\n
+    <span class="ic-stats-meta">\n
+    <span class="ic-stats-unit">days</span>\n
+    <span class="ic-stats-date">${datesCurrent}</span>\n
+    </span>\n
+    </span>\n
+    </span>\n
+    </div>`,
+    };
+  }
+
+  renderIsometrics = () => {
+    if (this.context) {
+      const obelisk = require("obelisk.js");
+      // Create a canvas 2D point for pixel view world
+      let point = new obelisk.Point(70, 70);
+      // Create view instance to nest everything
+      // Canvas could be either DOM or jQuery element
+      let pixelView = new obelisk.PixelView(this.context, point);
+
+      pixelView.clear();
+
+      // Get contributions of the selected year
+      let contribData;
+
+      if (this.props.year) {
+        contribData = this.props.platformData.statistic.years.find(
+          (element) => element.year === this.props.year
+        );
+      } else {
+        contribData = this.props.platformData.statistic.current;
+      }
+
+      let contributions = contribData.calendar;
+
+      // Define basic variables
+      const size = 2 * Math.round(this.state.width / 80 / 2);
+
+      if (size <= 8) {
+        size = 8;
+      }
+
+      const maxHight = 100;
```

Comment:
Please add a newline.

Reply:
In general, we do not differentiate between const and let for line breaks.
Introducing this rule here would break the coding style of all other repositories.